solc/CommandLineInterface.cpp

```diff
@@ -542,7 +545,8 @@ Allowed options)",
 			"Switch to linker mode, ignoring all options apart from --libraries "
 			"and modify binaries in place."
 		)
-		(g_argMetadataLiteral.c_str(), "Store referenced sources are literal data in the metadata output.");
+		(g_argMetadataLiteral.c_str(), "Store referenced sources are literal data in the metadata output.")
+		(g_argStandardJSON.c_str(), "Process standard JSON input / output.");
```

Comment:
I thought the idea was to expose the json/io API on all implementations of the compiler, in order to get a unified interface and to avoid the problems we currently have with the filesystem.